packages/symantec_endpoint_security/data_stream/event/manifest.yml

```
@@ -0,0 +1,345 @@
+title: Symantec Endpoint Security Event
+type: logs
+streams:
+  - input: aws-s3
+    enabled: false
+    template_path: aws-s3.yml.hbs
+    title: Event logs
+    description: Collect Symantec Endpoint Security logs via AWS S3 input.
+    vars:
+      - name: collect_s3_logs
+        required: true
+        show_user: true
+        title: Collect logs via S3 Bucket
+        description: To collect logs via S3 bucket enable the toggle switch. By default, it will collect logs via SQS Queue.
+        type: bool
+        multi: false
+        default: false
+      - name: bucket_arn
+        type: text
+        title: "[S3] Bucket ARN"
+        multi: false
+        required: false
+        show_user: true
+        description: It is a required parameter for collecting logs via the AWS S3 Bucket.
+      - name: queue_url
+        type: text
+        title: "[SQS] Queue URL"
+        multi: false
+        required: false
+        show_user: true
+        description: URL of the AWS SQS queue that messages will be received from. It is a required parameter for collecting logs via the AWS SQS.
+      - name: access_key_id
+        type: password
+        title: Access Key ID
+        multi: false
+        required: false
+        secret: false
```

Lets make this `true`

```suggestion
        secret: true
```